packages/store/src/internal/lifecycle-state-manager.ts

```diff
@@ -99,6 +99,7 @@ export class LifecycleStateManager implements OnDestroy {
   }
 
   private _getStateContext(mappedStore: MappedStore): StateContext<any> {
+    // Question: abort controller is not gonna be available for lifecycle hooks.
```

Comment:
I think that the `abortSignal` could be available for lifecycle hooks.
For example, when we have a local state, the abort signal for the lifecycle hooks would be triggered when the state should be unloaded.